src/components/Checkbox/_Checkbox.scss

```diff
@@ -1,8 +1,70 @@
-.checkbox {
-  align-items: baseline;
-  display: flex;
+$checkbox-size: to-rem(22px);
+$checkmark-color: $black;
+$checkmark-color-error: $error;
 
-  &__input {
-    @include space-inline-xs;
+.input {
+  &__checkbox {
+    -webkit-appearance: none;
+    border: 2px solid $gray-dark;
+    border-radius: 3px;
+    display: inline-block;
+    height: $checkbox-size;
+    outline: none;
+    position: relative;
+    vertical-align: middle;
+    width: $checkbox-size;
+
+    &:before {
+      content: "";
+      display: block;
+      height: $checkbox-size;
+      left: 50%;
+      position: absolute;
+      top: 50%;
+      transform: translate(-50%, -50%);
+      width: $checkbox-size;
+    }
+
+    &:checked {
+      &:before {
+        background: url('data:image/svg+xml;utf8,<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="#{url-friendly-colour($checkmark-color)}" d="M8.795 15.875l-4.17-4.17-1.42 1.41 5.59 5.59 12-12-1.41-1.41-10.59 10.58z"/></svg>')
+          50% 50% no-repeat;
+      }
+    }
+
+    + label {
+      display: inline-block;
+      font-size: var(--font-size-0);
+      font-weight: 300;
+      margin-bottom: 0px;
+      margin-left: var(--space-xs);
+      vertical-align: middle;
+    }
+
+    &--error {
+      border-color: $error;
+
+      + label {
+        color: $error;
+      }
+
+      &:checked {
+        border-color: $error;
+
+        &:before {
+          background: url('data:image/svg+xml;utf8,<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="#{url-friendly-colour($checkmark-color-error)}" d="M8.795 15.875l-4.17-4.17-1.42 1.41 5.59 5.59 12-12-1.41-1.41-10.59 10.58z"/></svg>')
+            50% 50% no-repeat;
+        }
+      }
+    }
+
+    &:disabled {
```

Comment:
There are some descrepencies in the Figma file regarding the styling of various states of Checkbox and Radio.  I'd like to get that ironed out before moving forward.  I'll keep you posted.